doc/api/next_api_changes/behaviour.rst

```diff
@@ -155,3 +155,12 @@ support for it will be dropped in a future Matplotlib release.
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Previously, keyword arguments were silently ignored when no positional
 arguments were given.
+
+
+`get_minorticklabels` and `get_majorticklabels` now returns plain list
```

Comment:
you likely(?) need dotted names here (https://matplotlib.org/devel/documenting_mpl.html#referring-to-other-code) -- just adding a leading dot should do.
See also the note at the end of https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects.

Reply:
Thanks for these links :-)

Reply:
Completed it!